src/ctia/entity/note.clj

```diff
@@ -0,0 +1,125 @@
+(ns ctia.entity.note
+  (:require [ctia.domain.entities :refer [page-with-long-id un-store-page]]
+            [ctia.entity.note.schemas :as note-schemas]
+            [ctia.http.routes.common :as routes.common]
+            [ctia.http.routes.crud :refer [services->entity-crud-routes]]
+            [ctia.lib.compojure.api.core :refer [GET routes]]
+            [ctia.schemas.core :refer [APIHandlerServices]]
+            [ctia.store :refer [list-records]]
+            [ctia.stores.es.mapping :as em]
+            [ctia.stores.es.store :refer [def-es-store]]
+            [schema-tools.core :as st]
+            [schema.core :as s]))
+
+(def note-mapping
+  {"note"
+   {:dynamic false
+    :properties
+    (merge
+     em/base-entity-mapping
+     em/sourcable-entity-mapping
+     em/stored-entity-mapping
+     {:entity_id em/token
+      :content em/searchable-token
```

Comment:
Content should be text only.
We will not sort on `content`.

Reply:
done